Formula/ghostscript.rb

```diff
@@ -48,6 +48,8 @@ def install
       --without-x
     ]
 
+    ENV.append_to_cflags "-DPNG_ARM_NEON_OPT=0"
```

Comment:
This flag follows the following link https://github.com/ArtifexSoftware/ghostpdl/blob/3244573621b1daa2997fbea0c13ce7f891b42203/libpng/INSTALL#L322